tsconfig.json

```diff
@@ -1,11 +1,19 @@
 {
   "compilerOptions": {
+    "incremental": true,
     "strict": true,
-    "noEmit": true
+    "noImplicitAny": true,
+    "declaration": true,
+    "outDir": "typings",
+    "target": "es2017"
   },
   "include": [
-    "typings",
-    "spec/typescript/*",
-    "spec/typescript/**/*"
+    "typings/**/*",
```

Comment:
Since `outDir` is set to `typings`, does that mean that we'll put the generated js code there, and include it back here? Isn't this a problem? 👀

Reply:
Thank you, that's a fact:
```
error TS5055: Cannot write file '/Users/mathias.klippinge/src/mappersmith/typings/utils.d.ts' because it would overwrite input file.
```

Maybe we need a `tsconfig.build.json` that will not include `typings/**/*` but will instead simple emit typings to this folder.